### PR TITLE
Fix extract_from_see_also() with None values

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: jbi/runner.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ def execute_action(
     The value returned by the action call is returned.
     """
     log_context = RunnerLogContext(
-        bug=request.bug.dict(),
-        request=request.dict(),
+        bug=request.bug,
+        request=request,
```
Comment (leplatrem) [inline jbi/runner.py:31]: drive by 🙄

**Review by grahamalama — APPROVED** [review]

**Review by bsieber-mozilla — APPROVED** [review]

File: jbi/runner.py (hunk 1 above)
Comment (bsieber-mozilla) [reply jbi/runner.py:31]: Accident?

**Review by bsieber-mozilla — APPROVED** [review]
The check looks to be failing because of no Label, but there is a 'bug' label applied. The change looks good to me.

File: jbi/runner.py (hunk 1 above)
Comment (leplatrem) [reply jbi/runner.py:31]: 😄  Somewhere between #216 and #222/#219 since the PRs were done simultaneously

**Pull request merged**